proctor-common/build.gradle

```diff
@@ -0,0 +1,51 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '8.1.1'
+}
+
+indeedOss.activateFeature 'library'
+indeedLibrary.name = 'proctor-common'
+
+dependencies {
+    implementation project(':proctor-tomcat-deps')
+    implementation ('com.indeed:util-core:1.0.49-1cfb888') {
+        exclude group: "org.slf4j", module: "slf4j-api"
+    }
+    implementation ('com.indeed:util-varexport:1.0.49-1cfb888') {
+        exclude group: "org.slf4j", module: "slf4j-api"
+    }
+
+    implementation libs.springCore
+    implementation libs.guava
+    implementation libs.commonsLang
+    implementation libs.jsr305
+    implementation libs.jacksonDatabind
+    implementation libs.log4jApi
+    implementation libs.log4jCore
+    testImplementation('org.slf4j:slf4j-api:1.7.32')
+    testImplementation libs.junit
+    testImplementation libs.assertj
+    testImplementation libs.easymock
+    testImplementation libs.easymockclassext
+    testImplementation libs.annotationApi
+    testImplementation libs.mockito
+    testImplementation libs.jsr305
+}
+
+publishing {
+    publications {
+        mavenJavaCommon(MavenPublication) {
+            groupId = 'com.indeed'
+            artifactId = 'proctor-common' // use the name from ext['indeed.publish.name']
+            version = '9999-SNAPSHOT' // use whatever version makes sense for your project
+
+            from components.java
+        }
+    }
+}
+
+shadowJar {
+    archiveClassifier.set('')
```

Comment:
This is a hack and breaks gradle's task output wiring. It'll complain that both the shadowJar and jar tasks contain the same output file.

Reply:
The shadow Jar relocation is widely used when I looked on sourcegraph. May required a bigger discussion/effort for removing the shadow jar.

Reply:
Moved this to top level build.gradle. Realized this was the wrong place to have this as it was relocating all the files in proctor-common and my hack was fixing that, otherwise creates run time invalid bytecode issues.